### PR TITLE
Removing serve_static_file from controllers

### PR DESCRIPTION
**Pull request by jbarrett**
@nilnilnil @malbin @MariagraziaAlastra @jdorweiler @jagtalon @russellholt 

With this change, `DDGC` no longer serves static files. It also replaces `ddgc_web_server.pl`.

```
$ script/ddgc_dev_server.sh -h

Usage: script/ddgc_dev_server.sh [-p port] [-m] [-h]

Options:

 -p     Specify listen port. Default: 5000
 -m     Use local debug mail server on port 1025
 -h     Show this text
```

### CODE REVIEW

**Comment by jbarrett** [comment]
I don't care if this breaks stuff - anything depending on it should break.

Let the chips fall where they may.

That said, I would like to get it out today, and would appreciate help in evaluating it (on view currently)

**Comment by MariagraziaAlastra** [comment]
LGTM :+1:

**Comment by nilnilnil** [comment]
It also removed Roboduck?

File: templates/default.tx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <h1>Page Not Found</h1>
 
-<p class="mid">This is what HTTP protocols like to call a <strong>404 error</strong>.  There's nothing at this address except ... ROBODUCK!!!</p>
+<p class="mid">Sorry, this link was not found. Please report dead links on duck.co to ddgc@duckduckgo.com. Thank you.</p>
```
Comment (nilnilnil) [inline templates/default.tx]: Thank you!

**Comment by nilnilnil** [comment]
:+1: nits on roboduck being grouped in and updating that text.

**Pull request merged**